fmriprep/workflows/confounds.py

```diff
@@ -7,36 +7,46 @@
 Calculates frame displacement, segment regressors, global regressor, dvars, aCompCor, tCompCor
 '''
 from __future__ import print_function, division, absolute_import, unicode_literals
-
 from niworkflows.nipype.interfaces import utility
+from niworkflows.nipype.interfaces import fsl
 from niworkflows.nipype.algorithms import confounds
 from niworkflows.nipype.pipeline import engine as pe
+from niworkflows.nipype.interfaces.fsl import ICA_AROMA as aroma
```

Comment:
Oh, sorry. I didn't realize it was `aroma.ICA_AROMA` below. I was assuming it was just `pe.Node(aroma())` and didn't check. Let's keep this as it is.